File: src/m/geometry/planetradius.js
function planetradius(planet) {//{{{
	/**
	 * PLANETRADIUS - return planet radius according to planetary body name
	 *
	 * Usage:
	 *     radius = planetradius(planet);
	 *
	 * Examples:
	 * earthradius = planetradius('earth');
	 */

	let radius = 0;
	if (planet === 'earth') {
		radius = 6.371012e6;
	} else if (planet === 'europa') {
		radius = 1.5008e6;
	} else {
		error('planet type ' + planet + ' not supported yet!');
	}
} //}}}

File: src/m/geometry/planetradius.m
function radius=planetradius(planet)
%PLANETRADIUS - return planet radius according to planetary body name.
%
%   Usage:
%      radius=planetradius(planet);
%
%   Examples:
%      earthradius=planetradius('earth');

if strcmpi(planet,'earth'),
	radius=6.371012*10^6;
elseif strcmpi(planet,'europa'),
	radius=1.5008*10^6;
else 
	error(['planet type ' planet ' not supported yet!']);
end
